repose-aggregator/components/filters/saml-policy-translation-filter/src/main/scala/org/openrepose/filters/samlpolicy/SamlPolicyTranslationFilter.scala

```diff
@@ -236,10 +261,46 @@ class SamlPolicyTranslationFilter @Inject()(configurationService: ConfigurationS
 
     if (Option(configuration).map(_.getPolicyAcquisition.getCache.getTtl) != Option(newConfiguration.getPolicyAcquisition.getCache.getTtl)) {
       cache = CacheBuilder.newBuilder()
-                          .expireAfterWrite(newConfiguration.getPolicyAcquisition.getCache.getTtl, TimeUnit.SECONDS)
-                          .build(new CacheLoader[String, XsltExecutable]() {
-                            override def load(key: String): XsltExecutable = getPolicy(key)
-                          })
+        .expireAfterWrite(newConfiguration.getPolicyAcquisition.getCache.getTtl, TimeUnit.SECONDS)
```

Comment:
Yeah, but i like to line up the method calls when doing a builder. I don't know maybe I'm weird?